### PR TITLE
feat(dbaas): OpenSearch service type support

### PR DESCRIPTION
**Pull request by villevsv-upcloud**

### CODE REVIEW

File: internal/commands/database/index/list.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		rows = append(rows, output.TableRow{
+			index.IndexName,
+			index.CreateTime,
+			index.Health,
+			index.Status,
+			index.Docs,
+			index.NumberOfReplicas,
+			index.NumberOfShards,
+			index.ReadOnlyAllowDelete,
+			fmt.Sprintf("%d bytes", index.Size),
```
[the remark this answers is not in the document]
Comment (villevsv-upcloud) [reply internal/commands/database/index/list.go:48]: Couldn't find one but indeed a good idea to add it later. Task added.

File: internal/commands/all/all.go [hunk 2]
```diff
@@ -134,6 +135,11 @@ func BuildCommands(rootCmd *cobra.Command, conf *config.Config) {
 		commands.BuildCommand(databaseproperties.ShowCommand(i.serviceType, i.serviceName), typeCommand.Cobra(), conf)
 	}
```
Comment (kangasta) [inline internal/commands/all/all.go:137]: Could also add OpenSearch as subcommand into above properties command, if it has properties available

File: internal/commands/database/index/delete.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+)
+
+type deleteCommand struct {
+	*commands.BaseCommand
+	name string
+	completion.Database
+	resolver.CachingDatabase
+}
+
+// DeleteCommand creates the "kubernetes indices delete" command
```
Comment (kangasta) [inline internal/commands/database/index/delete.go:22]: `kubernetes indices` → `database index`

**Comment by kangasta** [comment]
Added comment about `database properties` (and outdated code-comment), otherwise LGTM!

File: internal/commands/all/all.go (hunk 2 above)
Comment (villevsv-upcloud) [reply internal/commands/all/all.go:138]: Added

File: internal/commands/database/index/delete.go (hunk 3 above)
Comment (villevsv-upcloud) [reply internal/commands/database/index/delete.go]: Fixed!

**Pull request merged**